packages/sdk/src/auth.ts

```diff
@@ -217,7 +217,9 @@ export class AuthManager {
    */
   public async handleRedirect(url?: string): Promise<RedirectLoginResult<any> | undefined> {
     try {
-      return this.authentication.handleRedirectCallback(url);
+      const result = await this.authentication.handleRedirectCallback(url);
+      await this.authentication.checkSession();
```

Comment:
this hydrates the auth0client with the session token so the elements can render.

Reply:
Wouldn't this need to check session before handling the redirect? or I am maybe not fully understanding the flow

Reply:
Not quite, `handleRedirectCallback` does the work to grab the code from the url, validate, and set the cookie

Reply:
Removed this one. The above explaination still holds, but this one was unnecesssary